caas/clientconfig/k8s.go

```diff
@@ -0,0 +1,131 @@
+package caas
+
+import (
+	"os"
+	"path/filepath"
+
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+
+	"github.com/juju/errors"
+	"github.com/juju/loggo"
```

Comment:
`github.com/juju/*` repos aside from `github.com/juju/juju` are considered external and can be grouped with the other repos above.